pom.xml

```diff
@@ -83,6 +83,7 @@
         <dep.commons.compress.version>1.26.2</dep.commons.compress.version>
         <dep.protobuf-java.version>3.25.5</dep.protobuf-java.version>
         <dep.snakeyaml.version>2.0</dep.snakeyaml.version>
+        <dep.jetty.version>9.4.56.v20240826</dep.jetty.version>
```

Comment:
@infvg Looks like this version too has CVEs https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-bom/9.4.56.v20240826
Why not move to something latest may be`12.0.14` or `12.0.15`?

Reply:
We're currently using `9.4.54.v20240208` which has 2 additional CVEs. The latest versions require a JDK upgrade so at most we can only resolve those 2 CVEs for now.

Reply:
Yes. Keeping the updated version compatible with the current one is preferred.

Reply:
> We're currently using `9.4.54.v20240208` which has 2 additional CVEs. The latest versions require a JDK upgrade so at most we can only resolve those 2 CVEs for now.

Can you point me to the 2 additional CVEs with current version?

Reply:
> 9.4.54.v20240208

Yeah sounds good to me. Let's create an issue about this upgrade to the latest which is completely free from CVEs and link to this PR so that we get back to this again once we have the Java upgrade done.

To document here,
Current version - https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-bom/9.4.54.v20240208 has 3 CVEs
Upgraded version - https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-bom/9.4.56.v20240826 gets it to 1